plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -66,6 +71,13 @@ protected void execute(IProgressMonitor monitor)
         getArchetypeDescriptor(), groupId, artifactId, version, packageName, properties,
         importConfiguration, progress.newChild(60));
 
+    for (IProject project : archetypeProjects) {
+      IFacetedProject facetedProject = ProjectFacetsManager.create(
+          project, true, progress.newChild(20));
```

Comment:
The progress needs to be updated: this will overflow (created with 100 ticks at the top).